Cargo.toml

```diff
@@ -12,14 +12,14 @@ edition = "2018"
 anyhow = ">=1.0"
 aes = { version = ">=0.8", optional = true }
 async-trait = { version = "0.1.61", optional = true }
-attestation_agent = { git = "https://github.com/confidential-containers/attestation-agent.git", rev = "80b84cf", optional = true  }
+attestation_agent = { git = "https://github.com/confidential-containers/attestation-agent.git", rev = "3531788a75def92bac6d0e1ab57444c7bda73781", optional = true  }
```

Comment:
There are a few more commits I think we want to pick up. I think we want the hash `aa1d3c510350cd2f2668aca374abba19e2b73b3f`